lighthouse-core/runner.js

```diff
@@ -368,6 +368,7 @@ class Runner {
       const ArtifactClass = require('./gather/computed/' + filename);
       const artifact = new ArtifactClass(computedArtifacts);
       // define the request* function that will be exposed on `artifacts`
+      // @ts-ignore - doesn't have an index signature, so can't be set dynamically.
       computedArtifacts['request' + artifact.name] = artifact.request.bind(artifact);
```

Comment:
`artifact` and `ArtifactClass` are already `any` here, and `computedArtifacts` already has a `TODO` to refactor above this method, so it seems fine to just ignore and keep the issues contained here than to make `computedArtifacts` have a generic index signature and have it leak into audits, etc.